### PR TITLE
CON-2647-tier-prices

### PR DESCRIPTION
**Pull request by Carsten89**

### CODE REVIEW

File: pages/Product/components/Header/components/Tiers/style.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+const price = css({
+  fontWeight: 500,
+}).toString();
+
+export default {
+  price,
+  tier,
+  wrapper,
+};
```
Comment (kernelkiller) [inline pages/Product/components/Header/components/Tiers/style.js]: No newline at the end of the file

File: pages/Product/components/Header/components/Tiers/style.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2018, Shopgate, Inc. All rights reserved.
+ *
+ * This source code is licensed under the Apache 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { css } from 'glamor';
```
Comment (kernelkiller) [inline pages/Product/components/Header/components/Tiers/style.js]: Please add an empty line here

**Comment by kernelkiller** [comment]
Could you please add a basic unit test for the component?

File: pages/Product/components/Header/index.jsx [hunk 3]
```diff
@@ -52,6 +54,9 @@ const ProductHeader = () => (
         <div className={styles.priceInfo}>
           <Price />
         </div>
+        <div className={styles.priceInfo}>
```
[the remark this answers is not in the document]
Comment (Carsten89) [reply pages/Product/components/Header/index.jsx]: done

File: pages/Product/components/Header/components/Tiers/style.js (hunk 1 above)
Comment (Carsten89) [reply pages/Product/components/Header/components/Tiers/style.js]: done

File: pages/Product/components/Header/components/Tiers/style.js (hunk 2 above)
Comment (Carsten89) [reply pages/Product/components/Header/components/Tiers/style.js]: done

File: pages/Product/components/Header/components/Tiers/index.jsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+import connect from './connector';
+import styles from './style';
+
+/**
+ * The Tier Price component.
+ * @param {Object} props The component props.
+ * @return {JSX}
+ */
+const Tiers = ({ price }) => {
+  if (!(price && price.tiers.length)) {
```
[the remark this answers is not in the document]
Comment (Carsten89) [reply pages/Product/components/Header/components/Tiers/index.jsx]: done

File: pages/Product/components/Header/components/Tiers/index.jsx [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import I18n from '@shopgate/pwa-common/components/I18n';
+import connect from './connector';
+import styles from './style';
+
+/**
+ * The Tier Price component.
```
[the remark this answers is not in the document]
Comment (Carsten89) [reply pages/Product/components/Header/components/Tiers/index.jsx]: done

**Pull request merged**